My proposed changes to the cnn-logger.

caechols opened the pull request.

adslaton commented on CHANGELOG.md:
```diff
@@ -1,86 +0,0 @@
-# CNN Logger ChangeLog
```
Why is this file being deleted?

caechols replied:
I deleted all of the cnn-logger files and copied over from my other repo. we can ad that guy back for the history. This was a total from scratch rewrite.

adslaton commented on .nvmrc:
```diff
@@ -1 +0,0 @@
-6.2.2
```
Keep this file and use v8.7.x

caechols replied:
it is done!

adslaton commented on package.json:
```diff
@@ -1,51 +1,30 @@
-    "lib": "./lib",
-    "test": "./test"
-  },
+  "version": "2.0.0",
```
Do not change the version just yet. We will bump once we merge into master.

caechols replied:
I will change that back

adslaton commented on package.json:
```diff
@@ -1,51 +1,30 @@
-    "lib": "./lib",
-    "test": "./test"
-  },
+  "version": "1.0.0",
```
Should be 1.5.0

adslaton commented on LICENSE:
```diff
@@ -0,0 +1,21 @@
+MIT License
```
We use the Apache license.

caechols replied:
cool ill get that changed.

niahmiah commented:
I can't tell what the intent of this change is. I do see that travisCI and linting were removed. Why would you want to do that?

caechols commented:
@niahmiah I rewrote the code from scratch and it was in another repo. When I was asked to make A PR to see if it could be used as a refreshed logger I deleted the files from the branch and copied mine in. I did not realize that any of these things were needed. I can add them back no problem. I have eslint installed in the project, so I am not sure why it did not get checked in, but I will fix that.

niahmiah commented:
cool, thx. Just to catch me up, can you describe the differences between what this provided before, and what your changes provide?

caechols commented:
@niahmiah I know we slacked about this, but I wanted to put an answer here as well. In the discussions I was a part of the desire was to make the logger simpler, and resolve some issues around logging to logz and command characters being added to the logs. This is what I attempted and also tried keeping the same interface / signature for using it as well as accommodating multiple ways of targeted logging.

adslaton commented:
Bringing this PR back to light.

caechols commented:
@adslaton Oh snap I am excited. ill be on look out for any comments or changes to make!

DrEnter commented:
This appears to be a significant breaking change.  Also appears to drop the major performance enhancements we added in the last version.

caechols commented:
Interesting, What is the breaking change, If you dont mind me asking @DrEnter ? Also on . The performance aspect Is this version lesser in terms of performance, or are changes made just gone? This was a nearly 100% rewrite to simplify the logger and support logzio directly, or atleast that was my goal.
